src/content/docs/reference/Buffers/texture_format.mdx

```diff
@@ -8,115 +8,138 @@ sidebar:
 
 import { Aside } from '@astrojs/starlight/components';
 
-Color attachments, such as [colortex](/reference/buffers/colortex) and [shadowcolor](/reference/buffers/shadowcolor) buffers, store values in a specific format. The default format is `RGBA` (which equates to `RGBA8` on most systems), however a shader pack can specify the format and precision of each color attachment.
+Color attachments such as [colortex](/reference/buffers/colortex) and [shadowcolor](/reference/buffers/shadowcolor) buffers, as well as [images](/reference/buffers/custom_images/) and [textures](/reference/buffers/custom_textures/), store values in a specific format.
 
-The format for any color attachement can be set using the [\<bufferName\>Format](/reference/constants/buffer_format) directive.
+The default format for color attachments is `RGBA` (which equates to `RGBA8` on most systems). However, a shader pack can specify the format using the [\<bufferName\>Format](/reference/constants/buffer_format) directive.
 
-The available texture formats and their meanings are described below:
+The available texture formats and their meanings are described below.
 
-## Normalized
-
-Normalized buffers can store "floating point" values in a fixed range, between 0.0 and 1.0 (inclusive).  The values are not actually stored as floating point internally, but as an integer value. However, the result is converted to a floating point during read/write to a color attachment using a normalized format, so the shader will read/write floating point values to the attachments, however any values outside of the range will be clamped to fit in the range.
-
-| 8-bit | 16-bit | mixed    |
-| ----- | ------ | -------- |
-| R8    | R16    | RGB5_A1  |
-| RG8   | RG16   | RGB10_A2 |
-| RGB8  | RGB16  |          |
-| RGBA8 | RGBA16 |          |
+> Cells marked with ~strikethrough~ are currently unsupported.
```

Comment:
Is there any point in including formats which aren't supported? Or is this because of the discussion of adding support for more formats? Either way, I feel that for now there's not much of a point in documented unsupported formats, they can always be added later if support is increased

Reply:
> Is there any point in including formats which aren't supported? Or is this because of the discussion of adding support for more formats? Either way, I feel that for now there's not much of a point in documented unsupported formats, they can always be added later if support is increased

I added them both for consistency and clarity (to show which image formats, bit depths, pixel formats and pixel types are associated, even one of them is unsupported).

You can still load custom raw textures with pixel formats and types associated with unsupported image formats, and image formats with unsupported pixel formats can still be used as colortex formats.

It also clearly shows which formats exist in OpenGL but aren't supported by Iris and makes it very easy to mark them as supported if they ever get added. I feel it would be less clear if we were to write items associated with unsupported items in a separate table or leave cells of unsupported items empty

Reply:
Okay that makes sense, I would make sure to describe at least some of that then, because it wasn't clear to me at first glance.